src/main/resources/xslt/outputs/fr/models.xsl

```diff
@@ -415,10 +436,17 @@
     </xd:doc>
     <xsl:template match="Bind//RowLoop | Bind//QuestionLoop" mode="model">
         <xsl:param name="source-context" as="item()" tunnel="yes"/>
+        <xsl:param name="instance-ancestor" tunnel="yes"/>
```

Comment:
Comment should be added to explicit the param instance-ancestor (building, usage,...)